### ISSUE TITLE
Subdomains (aliases) are not pointing to the user's account

### ISSUE THREAD

**Issue by jorgesat** [YunoHost-Apps/ssbroom_ynh#10]
### Describe the bug

The app lets the user create a subdomain for their account that looks like *alias.domain.tld*. However, this is not pointing to the account but rather to the yunohost login page.

### Context

- Hardware: *VPS bought online* 
- YunoHost version: 4.3.6.3
- I have access to my server: *Through SSH | through the webadmin *
- Are you in a special context or did you perform some particular tweaking on your YunoHost instance?: *no*

### Steps to reproduce

Install the app and log in using any SSB application. Create an alias for your account.

### Expected behavior

The subdomain (*alias.domain.tld*) links to the user's account.

**Comment by mhfowler** [comment]
ah thanks for raising this @jorgesat . 

I think the way yunohost handles domains, it would not be straightforward (or even possible) to use the subdomain alias feature in the yunohost package ... but I recall there might be another way to achieve similar functionality without subdomains in ssbroom... but need to look into it further

cc @staltz

**Comment by staltz** [comment]
Yeah I think `room.club/alias/bob` is the same endpoint as `bob.room.club`
